src/cr/cube/cubepart.py

```diff
@@ -239,6 +239,10 @@ class _Slice(CubePartition):
     dimensions which can be crosstabbed in a slice.
     """
 
+    # quantile of the normal cdf at .975 because the confidence
```

Comment:
Don't make me look up what `cdf` is. Spell it out and then show the abbreviation in parentheses, like `Common Denominator Frombus (CDF)`. Also, if you would, prefix comments with `--- ` to make them stand out from commented-out code. Like:

```python
# --- Quantile of the normal Cubic Dimension Fromulus (CDF) at .975 because the
# --- confidence interval is ±.025 on each side.
```

What are the units of the ±.025 value? Is that standard-deviations? Best to be explicit. Use these opportunities to educate the uninitiated reader.